src/main/resources/db/changes/08-create-shopping-carts-table.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: maksymmilian
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false

```

Comment:
For primary/foreign keys and unique columns, It's better to have named constraints